packages/dashboard/package.json

```diff
@@ -35,22 +36,25 @@
     "@babel/runtime": "^7.20.0",
     "@callstack/repack": "~3.4.0",
     "@react-native/eslint-config": "^0.74.0",
+    "@rnx-kit/align-deps": "^2.2.5",
     "@tsconfig/react-native": "^2.0.2",
     "@types/jest": "^29.2.1",
     "@types/react": "^18.0.24",
+    "@types/react-native-vector-icons": "^6.4.12",
     "@types/react-test-renderer": "^18.0.0",
     "@typescript-eslint/eslint-plugin": "^5.37.0",
     "@typescript-eslint/parser": "^5.37.0",
     "babel-jest": "^29.2.1",
     "babel-loader": "^9.1.0",
     "eslint": "^8.19.0",
     "jest": "^29.2.1",
-    "metro-react-native-babel-preset": "0.73.9",
+    "metro-react-native-babel-preset": "^0.73.9",
     "prettier": "^2.8.8",
-    "react-test-renderer": "18.2.0",
+    "react-test-renderer": "^18.2.0",
+    "super-app-showcase-sdk": "0.0.1",
     "terser-webpack-plugin": "^5.3.6",
     "typescript": "^4.8.4",
-    "webpack": "^5.75.0"
+    "webpack": "^5.74.0"
```

Comment:
interesting, why the downgrade? 🤔

Reply:
result after aligning, I picked a version from host instead of dashboard as source of truth

Reply:
no longer applicable since alignment of deps was removed